ml_engine/online_prediction/predict.py

```diff
@@ -0,0 +1,173 @@
+# Copyright 2016 Google Inc. All Rights Reserved. Licensed under the Apache
+# License, Version 2.0 (the "License"); you may not use this file except in
+# compliance with the License. You may obtain a copy of the License at
+# http://www.apache.org/licenses/LICENSE-2.0
+
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations under
+# the License.
+"""Examples of using the Cloud ML Engine's online prediction service."""
+from __future__ import print_function
+# [START import_libraries]
+import googleapiclient.discovery
+# [END import_libraries]
+
+
+# [START authenticating]
```

Comment:
We generally show constructing the service in each snippet instead of centralizing it. Every indirection adds cognitive load to the users.